File: geometry_and_trig/circle_area_circumference.cpp
// circle_area_circumference.cpp - Compute area and circumference of a circle from radius.
// Author: Scott

#include <iostream>
#include <cmath>
using namespace std;

int main() {
    double r = 1.0;
    string input;

    cout << "Enter radius (default 1.0): ";
    getline(cin, input);
    if (!input.empty()) r = stod(input);

    double area = M_PI * r * r;
    double circ = 2 * M_PI * r;

    cout << "Area = " << area << ", Circumference = " << circ << endl;
    return 0;
}

File: geometry_and_trig/circle_area_circumference.py
"""
circle_area_circumference.py
Compute the area and circumference of a circle.
Author: Scott
"""

import math

def circle_metrics(radius):
    area = math.pi * radius ** 2
    circumference = 2 * math.pi * radius
    return area, circumference

if __name__ == "__main__":
    try:
        r = float(input("Enter radius (default 1.0): ") or 1.0)
        area, circ = circle_metrics(r)
        print(f"Area ≈ {area:.4f}")
        print(f"Circumference ≈ {circ:.4f}")
    except ValueError:
        print("Invalid input.")
